v2/components/autocomplete/autocomplete.tsx

```diff
@@ -78,11 +79,27 @@ export const RenderAutocomplete = (
     const debouncedVal = useDebounce(props.value as string, 350);
 
     React.useEffect(() => {
+        const convertWildcardToRegex = (pattern: string): RegExp => {
+            const escapedPattern = pattern.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+            const wildcardPattern = escapedPattern.replace(/\\\*/g, '.*');
+            return new RegExp(wildcardPattern, 'i');
+        };
```

Comment:
Rolling our own function to convert a wildcard to regex worries me... I'd suggest using minimatch, which we already use in the Argo CD UI.